### ISSUE TITLE
Problems with pctnl

### ISSUE THREAD

**Issue by alrik11es** [danibrutal/Forker#5]
It's difficult to have the pctnl extension on every system.
So there should be an alternative.
- [ ] Prepare for windows environment
- [ ] Execute threads as shell executions

**Comment by danibrutal** [comment]
Hey @alrik11es , I am afraid at the moment pctnl is a requirement for this library, and about shell executions, I am not sure I would like to take that direction.
About Windows support, could you suggest any approach to face that ?

Thanks :)

**Comment by alrik11es** [comment]
The thing with shell executions is that you have a delay implied and is not a real fork. So I'm with you that could have a major development impact. I guess noticing by the way that the newest versions of PHP will have forking support.

**Comment by danibrutal** [comment]
Yeah, I think the only possibility is using threads. That can be a good improvement of the package.
Wanna contribute ? :)

**Comment by alrik11es** [comment]
Maybe, just give me some time.
